crates/astria-sequencer/src/app.rs

```diff
@@ -139,7 +139,12 @@ impl App {
             .await
             // TODO: explain why this is an invariant of the system
             .expect("block height should be set");
-        info!(?tx, ?height, "executed transaction");
+        info!(
+            ?tx,
+            ?height,
+            sender = tx.signer_address().to_string(),
```

Comment:
No need for the extra to_string as it already impls Display:
```suggestion
            sender = %tx.signer_adress(),
```